lib/jsonapi/utils/list.ex

```diff
@@ -0,0 +1,50 @@
+defmodule JSONAPI.Utils.List do
+  @moduledoc """
+  Utility functions to build a list from a map in a different format than the one returned by Map.to_list/2
+  """
+
+  @doc """
+  Transform a map es. %{key1: value1, key2: value2} into a list of two element tuple.
```

Comment:
"es"?

Reply:
"Transforms a Map into a List of Tuples that can be converted into a query string via URI.encode_query/1"